src/__internal__/label/label.style.js

```diff
@@ -1,55 +1,51 @@
 import styled, { css } from "styled-components";
 import PropTypes from "prop-types";
-import BaseTheme from "../../style/themes/base";
 
 const LabelStyle = styled.label`
-  color: ${({ theme }) => theme.text.color};
+  color: var(--colorsYin090);
   display: block;
-  font-weight: 600;
+  font-weight: 600; //TODO: (tokens) use token var(--fontWeights500)
```

Comment:
Why can't we do this now?

Reply:
In DS we use 3 font-weight variants: (400, 500, 700). In this step we replace the theme with tokens only. The current use of tokens causes significant visual changes in thickness, and this is something we want to avoid. I've given a TODO here to introduce font-weight tokens in the next iteration, when we adjust the components for the new DS.

Reply:
In DS we use 3 font-weight variants: (400, 500, 700). In this step we replace the theme with tokens only. The current use of tokens causes significant visual changes in thickness, and this is something we want to avoid. I've given a TODO here to introduce font-weight tokens in the next iteration, when we adjust the components for the new DS.

Reply:
Ok I'm happy with that 👍🏻